content/docs/cve/cveTeams.md

```diff
@@ -1,55 +1,57 @@
-- Click on **+Add New Team** in the middle upper area below the ExamTools page header.
-- **Add New Team** window should pop-up with four fields for you to define.
+- Click on **Team Accounts** on the left navigation in Exam Tools.
+- Click on **+ New Team Account** in the middle upper area below the ExamTools page header.
+- **New Team Account** window should pop-up with four fields for you to define.
 - **Team Identifier**  (Team ID) - A team is identified on the schedule by a "team identifier" which starts with a letter, is 3-10 characters long, and can contain letters, numbers, and hyphens.  Any single word, call sign or acronym can be used to uniquely identify the team. ex. UVARC, NT3ST-R, etc.  Enter the call sign of the session manager, if you are not planning on having a team ID.
-- **Team Name** - the team’s name or group associated with the team
+- **Team Name** - the team’s name or group associated with the team account
 - **Contact E-mail Address** - a valid email address associated with the administrator (owner) of the team
-- **VEC** (Which of your VEC credentials should this team be allowed to use to schedule exams?) - Select an available VEC from the drop-down menu.  The individual setting up the team will have the option of selecting from the VECs for which they are a session manager/team lead/liaison in ExamTools.  Select all that apply to this team. 
+- **VEC** (Which of your VEC credentials should this team be allowed to use to schedule exams?) - Select an available VEC from the drop-down menu.  The individual setting up the account will have the option of selecting from the VECs for which they are a session manager/team lead/liaison in ExamTools.  Select all that apply to this account. 
 - When done, click **Save** to exit.
 
-## Team List : To edit or view teams
+## Team Accounts : To edit or view team accounts
 
- * From the **Team List**, locate the team you wish to view and click on the **Manage Team** button which will allow you to view all the fields and setting for the team.
+ * Within **Team Accounts**, locate the account you wish to view and click on the **Manage Account** button which will allow you to view all the fields and setting for the account.
  *  **Team Identifier:**  - Used by the applicants to login and join the exam; also used in permalinks for the session and the team on HamStudy.
  *  **Team Name:** - Displays along with the team identifier in the session listing on HamStudy.
- *  **E-mail Address:** - Should contain a valid email address for the administrator of the team.
- *  **Team Website:** - Enter the url (i.e. http://…) for the website for the club or group associated with this team. Leave blank if you do not have one.
- *  **My team permissions:** - The personal permissions settings for the team you are viewing. (not editable and set by the owner/co-owner of the session)
+ *  **Contact E-mail Addresses:** - Should contain a valid email addresses for the administrators of the account.
```

Comment:
@taxilian The should not be pluralized unless it allows entering multiple email addresses.
```suggestion
 *  **Contact E-mail Address:** - Should contain a valid email address for the administrator(s) of the account.
```

Reply:
I see you can add additional addresses *after* creating the team on the "manage" page. Perhaps the initial "New Team Account" modal dialog should allow adding multiple email addresses as well or change the wording.

Reply:
@hparkersd (these comments are for Richard to change the ET UI)

Reply:
Let's wait to see if @taxilian decides to change the ET code or not.